### DISCUSSION TITLE
WLgate support in Docker version of Wavelog

### DISCUSSION THREAD

**Discussion by berrySkies** [wavelog/wavelog#956] [WL Gate]
Hi,
I've successfully installed Wavelog using the docker container. I am now trying to get WL-Gate working so I can use flrig to interface with my radio (Thetis/Hermes Lite 2). I am struggling on how to setup HTTPS access to Wavelog so the API works. Are there any instructions on how to do this in the docker installation please?
Thanks Alan VK2ATW

**Comment by HB9HIL** [comment]
Hello @berrySkies 

I would recommend to use a Reverse proxy which points to your docker installation. There are plenty of tutorials in the www which are great. For example Nginx Proxy Manager is great for the start.
